src/common/shorters/bans.ts

```diff
@@ -0,0 +1,84 @@
+import type {
+	APIBan,
+	RESTGetAPIGuildBansQuery,
+	RESTPostAPIGuildBulkBanJSONBody,
+	RESTPutAPIGuildBanJSONBody,
+} from 'discord-api-types/v10';
+import { BaseShorter } from './base';
+import { GuildBan } from '../../structures/GuildBan';
+
+export class BanShorter extends BaseShorter {
+	/**
+	 * Bulk creates bans in the guild.
+	 * @param guildId The ID of the guild.
+	 * @param body The request body for bulk banning members.
+	 * @param reason The reason for bulk banning members.
+	 */
+	async bulkCreate(guildId: string, body: RESTPostAPIGuildBulkBanJSONBody, reason?: string) {
+		return await this.client.proxy.guilds(guildId)['bulk-bans'].post({ reason, body });
```

Comment:
```suggestion
	const bans = await this.client.proxy.guilds(guildId)['bulk-bans'].post({ reason, body });
	for (const id of bans.banned_users) this.client.cache.members?.removeIfNI('GuildBans', id, guildId);
	return bans;
```

Reply:
Since this exists in the other functions, I see it appropriate to add it here as well.

Reply:
done